kernel/kernel-util/src/main/java/org/sakaiproject/util/DateFormatterUtil.java

```diff
@@ -94,18 +94,17 @@ public static Date parseISODate(final String inputDate) {
 	 * 			If throws a parse exception then returns the SHORT format by default (MM/dd/yyyy hh:mm a)
 	 */
 	public static String format(Date inputDate, String format, Locale locale) {
-		SimpleDateFormat formatter = null;
-
 		if(inputDate == null){
 			return null;
 		}
 
 		try {
-			formatter = new SimpleDateFormat(format, locale);
-			return formatter.format(inputDate);
+			return new SimpleDateFormat(format, locale)
+					.format(inputDate);
 		} catch(Exception ex) {
-			formatter = (SimpleDateFormat) DateFormat.getDateTimeInstance(DateFormat.SHORT, DateFormat.SHORT, Locale.US);
-			return formatter.format(inputDate);
+			return DateFormat.getDateTimeInstance(DateFormat.SHORT, DateFormat.SHORT, Locale.US)
+					.format(inputDate)
+					.replace(",",""); // FIX JDK8 -> JDK9
```

Comment:
http://openjdk.java.net/jeps/252

Reply:
https://bugs.java.com/bugdatabase/view_bug.do?bug_id=8152154

Reply:
A workarround is do this replace trick.

Reply:
Here was some openjdk discussion 
http://mail.openjdk.java.net/pipermail/jdk9-dev/2017-April/005733.html

https://docs.oracle.com/javase/9/intl/internationalization-enhancements-jdk-9.htm#JSINT-GUID-9DCDB41C-A989-4220-8140-DBFB844A0FCA

Reply:
This might be a better change, to just specify the format we want rather than removing the comma.

-        return DateFormat.getDateTimeInstance(DateFormat.SHORT, DateFormat.SHORT, Locale.ENGLISH);
+        return new SimpleDateFormat("M/d/yy h:mm a", Locale.ENGLISH);

Reply:
IDK, seems that the portion of code that have the replace is for fallback cases.
Probably its better using the default, but didn't want to touch tests while I'm working on JDK9>=